protocols/v2/noise-sv2/src/responder.rs

```diff
@@ -277,6 +370,12 @@ impl Responder {
         ret
     }
 
+    /// Securely erases sensitive data in the responder's memory.
+    ///
+    /// Clears all sensitive cryptographic material within the [`Responder`] to prevent any
+    /// accidental leakage or misuse. It overwrites the stored keys, chaining key, handshake hash,
+    /// and session ciphers with zeros or other non-sensitive values. This function is typically
```

Comment:
same as https://github.com/stratum-mining/stratum/pull/1111/files#r1777159377

are we erasing by overwriting with zeros, or other values?

the current wording is ambiguous, we should make it clear